options/posix/generic/unistd-stubs.cpp

```diff
@@ -588,9 +588,19 @@ ssize_t pread(int fd, void *buf, size_t n, off_t off) {
 	return num_read;
 }
 
-ssize_t pwrite(int, const void *, size_t, off_t) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+ssize_t pwrite(int fd, const void *buf, size_t n, off_t off) {
+	ssize_t num_read;
```

Comment:
Nitpick: I guess this should be called `num_written`.

Reply:
Good catch!